src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -172,42 +172,46 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
                   selected="{{selectedItem}}"
                 >
                   <paper-item on-click="entityMoreInfo">
-                    <paper-icon-button
-                      icon="hass:information-outline"
-                      alt="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
-                      title="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
-                    ></paper-icon-button>
-                    [[localize('ui.panel.developer-tools.tabs.states.more_info')]]
+                    <paper-icon-item>
```

Comment:
this is good, but you should drop the `paper-item` now, as `paper-icon-item` is a `paper-item`.

Reply:
Okay, I removed them, much simpler now!